apis/v1alpha1/httproute_types.go

```diff
@@ -327,6 +332,47 @@ type GatewayObjectReference struct {
 	Name string `json:"name" protobuf:"bytes,2,opt,name=name"`
 }
 
+// RouteGatewayStatus describes the status of a route with respect to an
+// associated Gateway.
+type RouteGatewayStatus struct {
+	// Gateway is a reference to a Gateway object that is associated with
+	// the route.
+	Gateway GatewayObjectReference `json:"gateways" protobuf:"bytes,4,rep,name=gateways"`
+	// Conditions describes the status of the route with respect to the
+	// Gateway.  For example, the "Admitted" condition indicates whether the
+	// route has been admitted or rejected by the Gateway, and why.  Note
+	// that the route's availability is also subject to the Gateway's own
+	// status conditions and listener status, which see.
```

Comment:
I think this may have gotten cut off early?

Reply:
I'm not sure what you mean.  Are you referring to "which see"?  It's an admittedly unusual construct (similar to ["q.v."](https://en.wiktionary.org/wiki/q.v.)).  Maybe best just to delete it.